install_config/upgrades.adoc

```diff
@@ -687,16 +742,18 @@ endif::[]
 ifdef::openshift-origin[]
 ====
 ----
-# oc create -n openshift -f /usr/share/openshift/examples/image-streams/image-streams-centos7.json
-# oc create -n openshift -f /usr/share/openshift/examples/db-templates
-# oc create -n openshift -f /usr/share/openshift/examples/quickstart-templates
-# oc replace -n openshift -f /usr/share/openshift/examples/image-streams/image-streams-centos7.json
-# oc replace -n openshift -f /usr/share/openshift/examples/db-templates
-# oc replace -n openshift -f /usr/share/openshift/examples/quickstart-templates
+# oc create -n openshift -f roles/openshift_examples/files/examples/v1.1/image-streams/image-streams-centos7.json
+# oc create -n openshift -f roles/openshift_examples/files/examples/v1.1/db-templates
+# oc create -n openshift -f roles/openshift_examples/files/examples/v1.1/quickstart-templates
+# oc replace -n openshift -f roles/openshift_examples/files/examples/v1.1/image-streams/image-streams-centos7.json
+# oc replace -n openshift -f roles/openshift_examples/files/examples/v1.1/db-templates
+# oc replace -n openshift -f roles/openshift_examples/files/examples/v1.1/quickstart-templates
```

Comment:
Hmm, if both origin and enterprise we're assuming they're using openshift-ansible-roles package then they both need to point to /usr/share/ansible/openshift-ansible/roles/openshift_examples/files/examples/